rio/src/RIO/Prelude/Reexports.hs

```diff
@@ -215,6 +216,7 @@ module RIO.Prelude.Reexports
   , Prelude.show
   , Prelude.snd
   , Prelude.subtract
+  , Prelude.toEnum
```

Comment:
Unfortunately this function is partial, so we wouldn't want to expose it from `RIO`. As a result, I think it also makes sense to not export `fromEnum`. Perhaps a `RIO.Partial` module would make sense for these and others?

Reply:
I'd say putting `fromEnum` in partial simply because `toEnum` is partial would feel awkward and forced. I'd say just separate them. What else could go in `RIO.Partial` that might be useful?

Reply:
Maybe `read` would be a good fit. For that matter, we could have `head` and `tail`, but I'd imagine those would go better in `RIO.List.Partial`.

Reply:
What about `fromJust`?

Reply:
That could work too, not that I'm a big fan of the function 😛